robot/basestation/static/js/science.js

```diff
@@ -143,11 +164,13 @@ const drawWheel = (ctx, w, h, r, n, a) => {
   ctx.font = '18px serif'
   ctx.fillStyle = 'black'
   ctx.textAlign = 'center'
+  let indeces = [0, 3, 2, 1]
```

Comment:
it's "indices", but now I'm being really pedantic

Reply:
Fixed